claim/gql_mutations.py

```diff
@@ -264,6 +264,11 @@ class ClaimInputType(OpenIMISMutation.Input):
     items = graphene.List(ClaimItemInputType, required=False)
     services = graphene.List(ClaimServiceInputType, required=False)
 
+    patient_outcome = graphene.String(max_length=250, required=False)
+    visit_reason = graphene.String(max_length=500, required=False)
```

Comment:
There is already an Explanation field by default.
I'm not really sure to understand the difference here.